package.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "openfga",
+  "version": "0.0.0",
+  "private": true,
+  "description": "A high performance and flexible authorization/permission engine built for developers and inspired by Google Zanzibar ",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openfga/openfga.git"
+  }
+}
```

It's a real shame that this CLI requires a `package.json` file at all. It's required because it needs to determine the repo url. In my testing I discovered that having the below was suffificent:

```suggestion
{
  "_": "package.json file exists only for kacl changelog linting",
  "repository": {
    "url": "https://github.com/openfga/openfgxa"
  }
}
```

But would be better if we didn't have that at all. Especially because AFAICT, it isn't even being used.